source/libraries/ContractInterfaces.ts

```diff
@@ -0,0 +1,1075 @@
+// THIS FILE IS AUTOMATICALLY GENERATED BY `generateContractInterfaces.ts`. DO NOT EDIT BY HAND'
```

Comment:
Do we gain anything from generating this file?  We do it in Augur so we can use it for our integration tests, but in order to do that we need to make the file accessible.  In this case, we are just building a docker image and having this file inside the docker image isn't terribly helpful.  Also, I think our stuff is going to always call _through_ our own contract and never directly talk to Maker/Oasis, so I don't think we need their stuff in TS form.

Consider not generating this file as part of the build process (don't call `GenerateContractInterfaces.js`)

Reply:
We will use it during the deploy process, in a future PR

Reply:
Ah, because Maker deployment requires calling Maker contract functions.  OK.  👍 on this then.